sap-2-tier-marketplace-image/shared/os-disk-parts.json

```diff
@@ -100,7 +101,7 @@
             "uri": "[parameters('userImageVhdUri')]"
           },
           "vhd": {
-            "uri": "[concat('http://',parameters('userImageStorageAccount'),'.blob.core.windows.net/', resourceGroup().name, '/', parameters('vmName'), '-', variables('osDiskName'))]"
+            "uri": "[concat('http://',parameters('userImageStorageAccount'),'.blob.core.windows.net/', variables('resourceGroupLower'), '/', parameters('vmName'), '-', variables('osDiskName'))]"
```

Comment:
resourceGroup names can contain other chars/sequences not allowed in storage container names - can we just remove this and hardcode a container name (e.g. /vhds/...) - seems like that will work fine with this sample.